SWIFT_PACKAGE_MANAGER.md

```diff
@@ -27,9 +27,9 @@ There is a known Xcode bug, reported in [this GitHub issue](https://github.com/b
 
 To use the `BraintreeDataCollector` library, you must also check the box for `KountDataCollector`.
 
-To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreeThreeDSecure`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
+To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
 
-To use the `BraintreeThreeDSecure` library, you must also check the box for `CardinalMobile`. _(This only applies to versions greater than 5.2.0. See [instructions below](#braintreethreedsecure) if you are using versions 5.0.0 to 5.2.0.)_
+To use the `BraintreeThreeDSecure` library, you must also check the boxes for `CardinalMobile` and `PPRiskMagnes`. _(This only applies to versions greater than 5.2.0. See [instructions below](#braintreethreedsecure) if you are using versions 5.0.0 to 5.2.0.)_
```

Comment:
I missed that PPRiskMagnes is also required for 3DS when I opened the previous PR. Fixing that here.

Reply:
If we revert the change on the line above, we can get rid of this change

Reply:
Yeah, I'm just worried that someone would read "To use the `BraintreeThreeDSecure` library, you must also check the boxes for `CardinalMobile`" as meaning that you **only** need to include `CardinalMobile` and miss the fact that you also need `PPRiskMagnes`.

Reply:
Yeh - that is fair. But we can keep `BraintreeThreeDSecure` in the list above still.

Reply:
Sure, that sounds good 👍